webapp/package.json

```diff
@@ -49,11 +49,11 @@
     "node-sass": "3.4.2",
     "sass-loader": "3.2.0",
     "style-loader": "0.13.0",
-    "webpack": "2.1.0-beta.4"
+    "webpack": "webpack/webpack#master"
```

Comment:
We might want this set to a specific commit number just in case something breaks with webpack again. 0/5 on that though